mailpoet/lib/WooCommerce/Helper.php

```diff
@@ -223,4 +223,23 @@ public function getLatestCoupon(): ?string {
 
     return $coupon ? $coupon->post_title : null; // phpcs:ignore Squiz.NamingConventions.ValidVariableName.MemberNotCamelCaps
   }
+
+  /**
+   * Check whether the current request is processing a WooCommerce checkout.
+   * Works for both the normal checkout and the block checkout.
+   *
+   * This solution is not ideal, but I cheched with a few WooCommerce developers,
```

Comment:
No need to change but a tiny typo

```suggestion
   * This solution is not ideal, but I checked with a few WooCommerce developers,
```

Reply:
Thanks, I committed your suggestion.